packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-utils.tsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import { AUTH_STATUS_CODES, isNavigationFromDerivGO, isNavigationFromP2P } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+import VerificationStatusActionButton from '../../../Components/verification-status-action-button';
+
+export const getFilesDescriptions = () => ({
+    title: (
+        <Localize i18n_default_text='We accept only these types of documents as proof of your address. The document must be recent (issued within last 6 months) and include your name and address:' />
+    ),
+    descriptions: [
+        {
+            key: 'utility_bill',
+            value: <Localize i18n_default_text='Utility bill: electricity, water, gas, or landline phone bill.' />,
+        },
+        {
+            key: 'financial_legal_government_document',
+            value: (
+                <Localize i18n_default_text='Financial, legal, or government document: recent bank statement, affidavit, or government-issued letter.' />
+            ),
+        },
+        {
+            key: 'home_rental_agreement',
+            value: <Localize i18n_default_text='Home rental agreement: valid and current agreement.' />,
+        },
+    ],
+});
+
+type TAuthStatus = {
+    needs_poi?: boolean;
+    is_submitted?: boolean;
+};
+
+export const getPOAStatusMessages = (
+    status: typeof AUTH_STATUS_CODES[keyof typeof AUTH_STATUS_CODES],
+    auth_status?: TAuthStatus,
+    should_show_redirect_btn?: boolean
+) => {
+    const is_redirected_from_platform = isNavigationFromP2P() || isNavigationFromDerivGO();
+
+    const pendingButton = (onClick?: () => void, platform_name?: string) => {
+        if (!auth_status?.needs_poi) {
+            if (should_show_redirect_btn)
+                return (
+                    <VerificationStatusActionButton
+                        button_text={
+                            <Localize i18n_default_text='Back to {{platform_name}}' values={{ platform_name }} />
+                        }
+                        onClick={onClick}
```

Comment:
onClick function is optional. is this expected?

Reply:
yes, because for some of the statuses we don’t need action button